pkg/autoscaler/scaling/multiscaler.go

```diff
@@ -321,7 +333,20 @@ func (m *MultiScaler) tickScaler(ctx context.Context, scaler UniScaler, runner *
 		return
 	}
 
-	if runner.updateLatestScale(desiredScale, excessBC) {
+	var (
+		removalCandidates []string
+		err               error
+	)
+
+	if runner.decider.Spec.EnableGracefulScaledown {
+		removalCandidates, err = scaler.PrepareForRemoval(ctx, desiredScale)
+		if err != nil {
+			logger.Errorf("Cannot scale: %w", err)
+			return
```

Comment:
Instead of returning here, what do you think of falling back to non-graceful scaledown and pass in nil for `RemovalCandidates`, since graceful scaledown is an optional feature?

Reply:
yeah thats a good idea.